### PR TITLE
feature/atoms - Atoms styles, remaining variables values created, als…

### PR DESCRIPTION
**Pull request by WS-Jedp**
…o Button component and his styles

### CODE REVIEW

File: DesignSystem/src/styles/components/Buttons.scss [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+  text-align: center;
+  padding: $space-elements-small;
+  border: $color-primary-white 1px solid;
+  background-color: $color-primary-black;
+  color: $color-primary-white;
+  font-size: $font-size-p;
+  transition: 0.3s ease-in-out;
+  cursor: pointer;
+
+  &-primary-black{
```
Comment (onlyabdiel) [inline DesignSystem/src/styles/components/Buttons.scss]:
```suggestion
  &-primaryBlack{

```

File: DesignSystem/src/styles/components/Buttons.scss [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+    background-color: $color-primary-black;
+    color: $color-primary-white;
+
+    &:hover{
+      box-shadow: 0 0 12px rgba(255,255,255,0.6);
+    }
+  }
+
+  // -------Secondary Button -------
+  &-primary-white{
```
Comment (onlyabdiel) [inline DesignSystem/src/styles/components/Buttons.scss]:
```suggestion
  &-primaryWhite{

```

File: DesignSystem/src/styles/components/Buttons.scss [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+    background-color: $color-primary-white;
+    border: none;
+    color: $color-primary-black;
+
+    &:hover{
+      box-shadow: 0 0 12px rgba(12,12,12,0.3);
+    }
+  }
+
+  &-secondary-gray{
```
Comment (onlyabdiel) [inline DesignSystem/src/styles/components/Buttons.scss]:
```suggestion
  &-secondaryGray{

```

File: DesignSystem/src/styles/components/Buttons.scss [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+    background-color: $color-secondary-gray;
+    border: none;
+    color: $color-primary-black;
+
+    &:hover{
+      box-shadow: 0 0 12px rgba(12,12,12,0.3);
+    }
+  }
+
+  &-secondary-blue{
```
Comment (onlyabdiel) [inline DesignSystem/src/styles/components/Buttons.scss]:
```suggestion
  &-secondaryBlue{

```

File: DesignSystem/src/styles/components/atoms.scss [hunk 5]
```diff
@@ -0,0 +1,67 @@
+@import '../variables.scss';
+
+// Headings, sizes and weights
+.special-title {
```
Comment (onlyabdiel) [inline DesignSystem/src/styles/components/atoms.scss]:
```suggestion
.specialTitle {

```

**Review by onlyabdiel — COMMENTED** [review]
There is some comments for the class names

**Comment by WS-Jedp** [comment]
I quit!

**Review by onlyabdiel — APPROVED** [review]

**Pull request merged**